ReactNativeClient/lib/models/Note.js

```diff
@@ -85,13 +85,19 @@ class Note extends BaseItem {
 		if (body && body.length) {
 			const lines = body.trim().split('\n');
 			let output = lines[0].trim();
-			// Remove the first #, *, etc.
-			while (output.length) {
-				const c = output[0];
-				if (['#', ' ', '\n', '\t', '*', '`', '-'].indexOf(c) >= 0) {
-					output = output.substr(1);
-				} else {
-					break;
+			if (output.match(/\[.+\]\(.+\)/)) {
```

Comment:
I think this is not going to work for images where tag starts with !

Reply:
Yes, I'll just handle that case momentarily